.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Issue summary
```

Comment:
why is there an issue template in this PR?

Reply:
The issue template is added to remind users to paste in logs if applicable